tests/testsuite.src/run_fundamental.at

```diff
@@ -6977,6 +6977,80 @@ AT_CHECK([$COBCRUN_DIRECT ./prog], [0],
 AT_CLEANUP
 
 
+AT_SETUP([BLANK WHEN ZERO with NUMERIC-EDITED])
+AT_KEYWORDS([fundamental])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. prog.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01  W-X  PIC -ZZZBZZ9,99 BLANK WHEN ZERO.
+       01  W-Y  PIC  ZZZBZZ9,99 BLANK WHEN ZERO.
+       01  W-Z  PIC +ZZZBZZ9,99 BLANK WHEN ZERO.
+       01  J    PIC S9(8)  COMP-5.
```

Comment:
Please replace that source J by four source fields:

```cobolfree
01 L  PIC S9(8) SIGN LEADING.
01 T  PIC S9(8) SIGN TRAILING.
01 LS PIC S9(8) SIGN SEPARATE LEADING
01 LT PIC S9(8) SIGN SEPARATE TRAILING
```

as this then covers all available cases.

Reply:
Done.